docs/_packages/web_api.md

```diff
@@ -538,18 +520,20 @@ const result = await web.files.uploadV2({
       filename: 'logo-sm.png',
     },
   ],
- });
- ```
+});
+```
 
-### Handling binary data
-Several methods, `files.uploadV2`, `files.upload (legacy)`, and `users.setPhoto`, allow you to upload a file. In Node, there are a few
+Notice a `thread_ts` even snuck into that last example just for those threaded uploaded needs.
+
+**Handling binary data:**
+
+Several methods, `filesUploadV2`, `files.upload (deprecated)`, and `users.setPhoto`, allow you to upload a file. In Node, there are a few
 ways you might be dealing with files, or more generally, binary data. When you have the whole file in memory (like when
 you've just generated or processed an image), then in Node you'd have a `Buffer` that contains that binary data. Or,
```

Comment:
While we're here, maybe we can link to node's docs on `Buffer`s and `ReadStream`s?

https://nodejs.org/api/fs.html#class-fsreadstream
https://nodejs.org/api/buffer.html